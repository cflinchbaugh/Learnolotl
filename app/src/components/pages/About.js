import React, { Component } from 'react';

class About extends Component {

    render() {
        return (
            <div>
                <h1>About</h1>
                <div>
                    Version: 0.0.0
                    <a target="_blank" rel="noopener noreferrer" href="https://github.com/cflinchbaugh/level-up">GitHub Repo</a>
                </div>
            </div>
        );
    }
}

export default About;