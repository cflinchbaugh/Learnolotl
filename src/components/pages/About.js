import React from 'react';

function About (props) {
    return (
        <div className='mdCard'>
            <p>Version: BETA</p>
            <p>
                <a target="_blank" rel="noopener noreferrer" href="https://github.com/cflinchbaugh/level-up">GitHub Repo</a>
            </p>
            <p>
                A passion project built primarily to assist with my own study of the Japanese language, but designed to be flexible and practical for several learning use-cases.<br/><br/>

                GitHub description: "A flexible, JSON-eating study-aid application, great for language learning and other memorization-based studying."<br/><br/>

                Built with ReactJS. Conceived, designed, architected, and implemented by <a target="_blank" rel="noopener noreferrer" href="https://www.linkedin.com/in/christopher-flinchbaugh/">me</a>.<br/><br/>
            </p>
        </div>
    );
}

export default About;