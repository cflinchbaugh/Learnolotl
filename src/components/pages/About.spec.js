import React from 'react';
import {shallow} from 'enzyme';
import About from './About';
import ShallowRenderer from 'react-test-renderer/shallow';

describe('About', () => {
    it('renders and matches snapshot', () => {
        const renderer = new ShallowRenderer();

        const aboutData = {
            
        }

        const tree = renderer.render(
            <About {...aboutData}/>
        );

        expect(tree).toMatchSnapshot();
    });

    
});