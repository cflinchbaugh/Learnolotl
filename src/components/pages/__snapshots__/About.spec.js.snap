// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders and matches snapshot 1`] = `
<div
  className="mdCard"
>
  <p>
    Version: BETA
  </p>
  <p>
    <a
      href="https://github.com/cflinchbaugh/Learnolotl"
      rel="noopener noreferrer"
      target="_blank"
    >
      GitHub Repo
    </a>
  </p>
  <p>
    A passion project built primarily to assist with my own study of the Japanese language, but designed to be flexible and practical for several learning use-cases.
    <br />
    <br />
    GitHub description: "A flexible, JSON-eating study-aid application, great for language learning and other memorization-based studying."
    <br />
    <br />
    Built with ReactJS. Conceived, designed, architected, and implemented by 
    <a
      href="https://www.linkedin.com/in/christopher-flinchbaugh/"
      rel="noopener noreferrer"
      target="_blank"
    >
      me
    </a>
    .
    <br />
    <br />
  </p>
</div>
`;
